code/modules/vehicles/vehicle.dm

```diff
@@ -117,7 +117,8 @@
 /obj/vehicle/attackby(obj/item/I, mob/user, params)
 	if(keytype && istype(I, keytype))
 		user << "Hold [I] in one of your hands while you drive \the [name]."
-
+	else
+		return ..()
 
 /obj/vehicle/Bump(atom/movable/M)
 	. = ..()
```

Comment:
The dmi below contains only sprite that are copies of doorfire.dm or something. The file is never used anywhere.